R/agglomerate.R

```diff
@@ -263,11 +371,14 @@ setMethod("agglomerateByRank", signature = c(x = "SingleCellExperiment"),
 #' @importFrom SingleCellExperiment altExp altExp<- altExps<-
 #' @export
 setMethod("mergeFeaturesByRank", signature = c(x = "SingleCellExperiment"),
-          function(x, ..., altexp = NULL, strip_altexp = TRUE){
-              .Deprecated(old="agglomerateByRank", new="mergeFeaturesByRank", "Now agglomerateByRank is deprecated. Use mergeFeaturesByRank instead.")
-              x <- agglomerateByRank(x, ..., altexp = altexp, strip_altexp = strip_altexp)
-              x
-          }
+            function(x, ..., altexp = NULL, strip_altexp = TRUE){
+                .Deprecated(old="agglomerateByRank", new="mergeFeaturesByRank",
+                            "Now agglomerateByRank is deprecated. 
+                            Use mergeFeaturesByRank instead.")
+                x <- agglomerateByRank(x, ..., altexp = altexp, 
+                                        strip_altexp = strip_altexp)
+                x
```

Comment:
Move these to deprecate.R

These functions will be removed. only agglomerateByRank, agglomerateByPrevalence and agglomerateByVariable will stay